trieste/utils/pareto.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from trieste.data import Dataset
+from trieste.type import TensorType
+
+
+def non_dominated_sort(datasets: Dataset) -> TensorType:
```

Comment:
since this only requires the observations, what about just passing it that?

Reply:
so just pass a tensor of observations?

Reply:
yes exactly, just as you've done